modules/virt-configuring-vm-live-migration-cli.adoc

```diff
@@ -24,7 +24,7 @@ $ oc edit vm <custom-vm> -n <my-namespace>
 
 [source,yaml]
 ----
-apiVersion: kubevirt.io/v1alpha3
+apiVersion: kubevirt.io/v1beta1
```

Comment:
@stu-gott this is still `kubevirt.io/v1alpha3`, right?

Reply:
Correct, it should still be `kubevirt.io/v1alpha3`